src/themes/appletv/theme.css

```diff
@@ -463,6 +463,10 @@ html {
     border-color: #00a4dc !important;
 }
 
+.card.show-focus:not(.show-animation) .cardBox:not(.visualCardBox) .cardScalable {
```

Comment:
Second line for "Poster/Thumb card" style.
You can change it using `Select view` in library.
```suggestion
.card.show-focus:not(.show-animation) .cardBox.visualCardBox,
.card.show-focus:not(.show-animation) .cardBox:not(.visualCardBox) .cardScalable {
```
![card-2](https://user-images.githubusercontent.com/56478732/86638214-92d0ef80-bfdf-11ea-806e-66a8d174ccc7.png)

Reply:
Thanks, I'll change it